app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def index
+    random_greeting = Message.find(rand(1..5))
```

Comment:
[OPTIONAL] Rails have build in randomizer. I think it would be better if you use it. This is my suggestion :)
```suggestion
    random_greeting = Message.all.sample
```

Reply:
Thank you for the feedback!!!